plugins/CoreHome/templates/_dataTable.twig

```diff
@@ -42,7 +42,8 @@
 
     <div class="reportDocumentation">
         {% if properties.documentation|default is not empty %}<p ng-bind-html="{{ properties.documentation|json_encode|e('html_attr') }}"></p>{% endif %}
-        {% if reportLastUpdatedMessage is defined and reportLastUpdatedMessage %}<span class='helpDate'>{{ reportLastUpdatedMessage|raw }}</span>{% endif %}
+        {% if properties.onlineGuideUrl|default is not empty %}<a href="?module=Proxy&action=redirect&url={{ properties.onlineGuideUrl|e('html_attr') }}" target="_blank" class="onlineGuide">{{ 'CoreHome_ReadMoreOnlineGuide'|translate }}</a>{% endif %}
```

Comment:
Is there a reason to use Proxy here instead of `<a rel="noreferrer noopener">` link?

Reply:
Would we also need to add `|safelink` filter? Might not be needed for our own links but I reckon cannot hurt?

Reply:
Can change that. Thought might be better to use the proxy so no third party plugin could add malicious here